python.wake

```diff
@@ -100,3 +107,44 @@ target installPipenvEnv pipDir =
   # Finally, run the plan and return tbe list of files installed in the venv.
   | runJob
   | getJobOutputs
+
+##############################################################################
+# Install the python environment associated with requirements.txt
+#
+# WARNING: When installing to the .venv/bin directory,
+#   pipenv creates scripts which embed the "realpath"
+#   of the Python binary. Under Wake, these paths contain
+#   explicit references to the Fuse directories.
+#
+#   So, pipenv creates invalid paths like the following:
+#       /workspace/.build/91/.venv/bin/python3.7
+#   when it should create the following path:
+#       /workspace/.venv/bin/python3.7
+#
+#   There will be an upcoming fix for this problem, but for now,
```

Comment:
Do we have an issue filed for this problem? Just curious, I understand this is just a copy/paste from another comment

Reply:
No, at least not externally, on any public repositories. This problem is not easily solvable under the current way the Wake FUSE runner works, since Wake moves files into temporary directories, and the Python build scripts seem to resolve the them into their absolute paths within the temporary directories. I'd like to solve this problem by not having Wake invoke the language-specific packaging setup code in the first place, which is part of the broader problem of creating isolated environments independent of Wake and Wit.